lib/msf/ui/console/history_manager.rb

```diff
@@ -0,0 +1,66 @@
+# -*- coding: binary -*-
+
+module Msf
+module Ui
+module Console
+
+class HistoryManager
```

Comment:
Unfortunately, because this is being called from a few locations within `Rex::` this should be moved into `Rex::`. Metasploit tries to ensure that the `Msf` namespace can call into `Rex` but not the inverse.

This would fit nicely under `Rex::Ui::Text::Shell` since that's already a module making it `Rex::Ui::Text::Shell::HistoryManager`.

Reply:
okay, I'll move it to `Rex::Ui::Text::Shell` :+1: